registry/internal/helm/deployer.go

```
@@ -0,0 +1,352 @@
+	return loader.Load(chartDir)
+}
+
+// writeTempValues writes values to a temporary file and returns its path
+// along with a cleanup func that deletes it.
+func writeTempValues(namespace string, values []byte) (path string, cleanup func(), err error) {
+	f, err := os.CreateTemp("/tmp/helm-values", fmt.Sprintf("harbor-%s-*.yaml", namespace))
+	if err != nil {
+		return "", func() {}, err
+	}
+	cleanup = func() {
+		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)